source/slang/hlsl.meta.slang

```diff
@@ -10125,6 +10125,27 @@ struct RayQuery <let rayFlagsGeneric : RAY_FLAG = RAY_FLAG_NONE>
         }
     }
 
+    __glsl_extension(GL_EXT_ray_query)
+    __glsl_extension(GL_EXT_ray_tracing_position_fetch)
+    __glsl_version(460)
+    [__NoSideEffect]
+    void GetIntersectionTriangleVertexPositions(out float3 positions[3])
```

Comment:
I suggest we just expose as `float3 GetIntersectionTriangleVertexPosition(int index)` to keep consistent with the existing HitTriangleVertexPosition function.

Reply:
There is a little problem by using this signature.
The glsl signature is 
```
void rayQueryGetIntersectionTriangleVertexPositionsEXT(rayQueryEXT q,
                                                               bool committed,
                                                               out vec3 positions[3]);
```

so if we define the signature as `float3 GetIntersectionTriangleVertexPosition(int index)`, we will call this function 3 times to get all vertices.